### DISCUSSION TITLE
Login Failure Without Errors

### DISCUSSION THREAD

**Discussion by GitRandomUsrname** [mitre/caldera#2981] [Q&A]
I have an RHEL server on which I installed all the pre-requirements. 

I cloned the git recursively without errors. I ran the 'pip3 install—r requirements.txt' and received a warning about how doing a pip install with root is generally a bad idea, but no errors. 

I ran python3 server.py --build and received a warning about babel-eslinit being deprecated, but all seemed pretty normal to me:
![image](https://github.com/mitre/caldera/assets/169851345/200f2d21-6ff5-4fb4-8b65-fe793724100c)

The problem is that from my admin box (external to the RHEL server), I can use a browser to open the login page via http://<RHEL IP>:8888/login, but when I use the username and password from the conf/local.yml file it does nothing. I don't see it providing an error or taking me to the console.  If I click refresh, I can briefly see the icon of a person in the upper left has changed from white to blue (or red, depending on the account I try to login as) but it almost immediately takes me back to the login screen.

 If I put the server in debug mode, I can see this:
![image](https://github.com/mitre/caldera/assets/169851345/8636c08f-3dc2-47a1-b339-6e890b3e697f)

I've tried modifying the local.yml file to set the host and the app.contact.http settings to have the RHEL server's IP address, but the behavior didn't change. I'm not sure how to progress further. Thanks for any help you can provide.

**Comment by ns0r** [comment]
This is a bug that has not been resolved

**Comment by jamestwebb** [comment]
Are there any known work-arounds for this? 

I seem to hit this issue on both local MacOS and Linux Installs using venvs.
